### PR TITLE
Adding mailed alerts.

### PR DESCRIPTION
**Pull request by michelle192837**

### CODE REVIEW

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -18,9 +18,10 @@ default_test_group:
   - configuration_value: Commit # Shows the commit number on column header
   - configuration_value: infra-commit
   num_columns_recent: 10
-  use_kubernetes_client: true # These two fields are depricated and should always be true
+  use_kubernetes_client: true # These two fields are deprecated and should always be true
```
Comment (krzyzacy) [inline testgrid/config/config.yaml]: :joy: thanks for correcting my terrible spelling!

File: testgrid/config/config.yaml [hunk 2]
```diff
@@ -3084,6 +3087,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-prow-canary
     code_search_url_template:
       url: https://github.com/kubernetes/test-infra/compare/<start-custom-0>...<end-custom-0>
+    dashboard_tab_alert_options:
+      alert_mail_to_addresses: michelle.192837+alerts@gmail.com
```
Comment (krzyzacy) [inline testgrid/config/config.yaml]: you are not likely to receive emails for this job... I'll find you a better(flakier) one

Comment (krzyzacy) [reply testgrid/config/config.yaml]: (try http://k8s-testgrid.appspot.com/master-upgrade#gce-1.7-master-upgrade-master), and add lusensjtu+alerts@gmail.com as well :-)

Comment (michelle192837) [reply testgrid/config/config.yaml]: Done?

**Comment by krzyzacy** [comment]
:-) now you want to fix your own test, then LGTM and squash

**Comment by krzyzacy** [comment]
whoops - plz ignore the bot :-)

**Comment by krzyzacy** [comment]
/lgtm
wooo! :raised_hands: 
(squash the second commit?)

**Comment by krzyzacy** [comment]
prime time!

**Pull request merged**